server/events.go

```diff
@@ -88,20 +90,32 @@ type ServerStatsMsg struct {
 
 // ConnectEventMsg is sent when a new connection is made that is part of an account.
 type ConnectEventMsg struct {
+	Schema string     `json:"schema"`
+	ID     string     `json:"id"`
```

Comment:
I understand the comment about the added size for each event, but not sure what is the alternative. Could we have codes for schemas here that gets "lookup" by the receiving application? (like a dictionary from code to URN)?

Reply:
These are primarily to help others, if someone sees these events in a  event router who may have consumed it via OpenFaaS via a webhook that may have consumed it via their JetStream adapter - you'll have lost all the context of what subjects were involved and so forth, so you need hints.  In those tools you can route different message types by this Schema and when someone/something looks at the schema they should be able to go to a URL, so it's obvious how to convert these strings to for example `https://nats.io/schemas/jetstream/api/v1/consumer_configuration.json`.  We could store there not just JSON Schema but other formats - avro etc - and it's easy to transform this to a NATS based Request() to resolve the schema too.

So they're unfortunately verbose and carry context and we can't maintain a lookup table since most of the time they aren't there to help us at all, they are for others who might get these events in systems quite far removed from us.

Reply:
I have some ideas here for future work that would allow us to have the best of both worlds possibly.